docs/victoriametrics-cloud/README.md

```diff
@@ -13,35 +12,16 @@ for further details.
 * [Quick Start](/victoriametrics-cloud/quickstart/) documentation.
 * [Try it now](https://console.victoriametrics.cloud/signUp?utm_source=website&utm_campaign=docs_overview) with a free trial.
 
+
 ## Use cases
-The most common use cases for VictoriaMetrics Cloud are:
-* Long-term remote storage for Prometheus metrics.
+VictoriaMetrics Cloud is for teams and organizations that deal with medium to high volume of metrics. The most common use cases for VictoriaMetrics Cloud are:
```

Comment:
can this create the false assumption that if you have small setup you don't use Cloud?

Reply:
That's a good point. I took the sentence from our product page. Rephrasing to: `VictoriaMetrics Cloud is designed for teams and organizations that handle any volume of metrics.`
I've created a PR for vmsite too: https://github.com/VictoriaMetrics/vmsite/pull/336